draft-ietf-mls-protocol.md

```diff
@@ -662,17 +662,113 @@ Cryptographic algorithms are indicated using the following types:
 
 ~~~~~
 enum {
+    /* Elliptic Curve Groups (ECDHE) */
```

Comment:
If we're going to decouple like this, then we should just incorporate the TLS algorithm identifiers by reference.